plugins/scaffolder-backend/api-report.md

```diff
@@ -429,6 +429,7 @@ export const createPublishGitlabMergeRequestAction: (options: {
   title: string;
   description: string;
   branchName: string;
+  sourcePath: string;
```

Comment:
```suggestion
  sourcePath?: string;
```

sourcePath should be made optional here, we have the same thing for `createPublishGitlabAction` above. I would update targetPath as well to also be optional